pom.xml

```diff
@@ -116,6 +142,8 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
+
+		<!-- //TODO do we need this? -->
```

Comment:
we don't need it now, but it's good to have it there in the long run when versions are upgraded